pyanaconda/kickstart.py

```diff
@@ -349,7 +349,7 @@ def execute(self, *args):
         if not flags.automatedInstall and self.fingerprint_supported:
             self._run(
                 "/usr/bin/authselect",
-                ["select", "sssd", "with-fingerprints", "--force"],
+                ["select", "sssd", "with-fingerprint", "with-silent-lastlog", "--force"],
```

Comment:
This seems to be safe. I suppose, that it is ok, that the option `with-silent-lastlog` will not be applied if `self.fingerprint_supported` is False.

Reply:
Since Anaconda only calls authselect automatically in this case and otherwise does not touch the configuration, it is ok.